.github/CODEOWNERS

```diff
@@ -3,4 +3,7 @@
 #  @department-of-veterans-affairs/frontend-review-group will be requested for
 # review when someone opens a pull request.
 
-*       @department-of-veterans-affairs/frontend-review-group
+* @department-of-veterans-affairs/frontend-review-group
+/src/applications/vaos/ @department-of-veterans-affairs/teams/va-gov-vaos
+*.scss @department-of-veterans-affairs/frontend-review-group
+*.css @department-of-veterans-affairs/frontend-review-group
```

Comment:
in CODEOWNERS, only one required approval is triggered on a PR at a time. Also, 

> the last matching pattern takes the most precedence

With this config, the FE review group is the default approval needed for any change to the repo.

Anything change the /app/vaos folder requires approval by the vaos group and not the FE review group.

Any change to scss or css require approval by the FE review group.

Reply:
@jbalboni :  fyi- we're using your group as an example. 

@department-of-veterans-affairs/frontend-review-group @gunsch how do you think we should get the rest of the teams groups set up? looks like nancy set up the vaos group so maybe this is already in progress.

Reply:
ah okay I found this: https://github.com/orgs/department-of-veterans-affairs/teams/vets-gov-write/teams

So I think we could ask teams to add "review groups" as suub groups to their contract groups. we should probably move the FE review group into this https://github.com/orgs/department-of-veterans-affairs/teams/vets-gov-write group

Reply:
I'd check with @nwalke about intent/administration of GH teams structure as that gets built out. I don't know enough about which GH teams we do/don't have already, particularly for GH teams that correspond to VFS teams.